hooks/useStakingData.ts

```diff
@@ -50,54 +50,59 @@ type VestingEntry = {
 	entryID: number;
 };
 
+type VestingClaimable = {
+	quantity: number;
```

Comment:
Seems like values which should be strings or BigNumbers?

Reply:
Yes. This should be `BigNumber`. The change has been made.